A/ArcadeLearningEnvironment/build_tarballs.jl

```diff
@@ -8,12 +8,20 @@ version = v"0.6.1"
 # Collection of sources required to complete build
 sources = [
     ArchiveSource("https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.6.1.tar.gz", "8059a4087680da03878c1648a8ceb0413a341032ecaa44bef4ef1f9f829b6dde"),
+    FileSource("http://www.atarimania.com/roms/Roms.rar", "4e35879fbd3da7d008f80f8d3a48360b9513859aa6c694164e67d5a82daca498"),
+    FileSource("https://raw.githubusercontent.com/mgbellemare/Arcade-Learning-Environment/v0.6.1/md5.txt", "218673cbeba56f3a7066293c259ae7b31ebece686bf3ff4ae4fe746e7d58a51e"),
```

Comment:
Can't this be a separate platform-independent artifact?

Reply:
Good idea! I tried to do so in #2930.